### PR TITLE
Define dockerfile for frontend

### PR DESCRIPTION
**Pull request by katerinapat**

### CODE REVIEW

**Review by droslean — COMMENTED** [review]
few comments!

File: cmd/dashboard/frontend/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
[the remark this answers is not in the document]
Comment (katerinapat) [reply cmd/dashboard/frontend/.dockerignore:1]: Removed it

File: cmd/dashboard/frontend/Dockerfile.prod [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# build environment
```
[the remark this answers is not in the document]
Comment (katerinapat) [reply cmd/dashboard/frontend/Dockerfile.prod]: `Dockerfile` should be enough for starting the frontend, this I'm removing `Dockerfile.prod`

**Comment by droslean** [comment]
I am going to merge this since the GitHub actions docker build already includes a DNS, so there will be no issue to build this.

**Pull request merged**